README.md

```diff
@@ -131,12 +131,13 @@ You need to set your environment by running the credentials. These will be run w
 . credentials/.email_server
 ```
 
-At the moment, when you run the tests, any HTTP request will be recorded once in `spec/cassettes`, and then will be replayed everytime you run the tests. So the first time that you run them, they will take long, but once they are recorded, they should run faster.
+At the moment, when you run the tests, any HTTP request will be recorded once in `spec/cassettes`, and then will be replayed everytime you run the tests. So the first time that you run them, they will take long, but once they are recorded, they should run faster. Also, the first time you run the tests, you need to be connected to the internet.
```

Comment:
Reworded:

Currently, when you run the tests, HTTP request will be recorded once in `spec/cassettes`, which are then replayed every time you run the tests. So the first time that you run them, they will take a while, but once they are recorded they will run faster. Also, the first time you run the tests, you need to be connected to the internet.